docs/references/phpdoc/tags/param.rst

```diff
@@ -62,11 +58,19 @@ Examples
     /**
```

Comment:
I'd prefix this example with `Here is a fully formed example, where all arguments are given param tags even when not strictly required.`

Then, below the example, I'd suggest a second example showing the minimum required:

```
    /*
     * Counts the number of items in the provided array.
     *
     * @param $items     Array structure to count the elements of.
     * @param $recursive Whether or not to recursively count elements in nested arrays.
     *
     * @return int Returns the number of elements.
     */
    function count(array $items, bool $recursive = false)
```